O21.Game/U95/Help.fs

```diff
@@ -28,29 +29,29 @@ let private style = function
 
 let private convertParagraphs (fonts: FontDescriptor[]) (bitmaps: int -> Texture) (items: IParagraphItem seq) = seq {
     let mutable currentFont = None
+
     for item in items do
         match item with
         | :? FontChange as fc ->
             let font = fonts[int fc.FontDescriptor]
             currentFont <- Some font
-        | :? ParagraphText as pc ->
-            yield DocumentFragment.Text(style currentFont, pc.Text)
-        | :? NewParagraph ->
-            yield DocumentFragment.NewParagraph
-        | :? Bitmap as b ->
-            yield DocumentFragment.Image(bitmaps <| int b.Number)
+        | :? ParagraphText as pc -> yield DocumentFragment.Text(style currentFont, pc.Text)
+        | :? NewParagraph -> yield DocumentFragment.NewParagraph
+        | :? Bitmap as b -> yield DocumentFragment.Image(bitmaps <| int b.Number)
         | _ -> failwith $"Unknown paragraph item: {item}"
 }
 
-let extractDibImageFromMrb(file: byte[]): Dib =
+let extractDibImageFromMrb (file: byte[]) : Dib =
     use stream = new MemoryStream(file)
     let file = MrbFile.Load stream
+
```

Comment:
Let's also set [`fsharp_blank_lines_around_nested_multiline_expressions`](https://fsprojects.github.io/fantomas/docs/end-users/Configuration.html#fsharp_blank_lines_around_nested_multiline_expressions) to `false` to avoid adding this newline?